src/platform/datapath_linux.c

```diff
@@ -1616,6 +1658,11 @@ CxPlatSocketCreateUdp(
         }
     }
 
+    Status = CxPlatSocketConfigureRss(&Binding->SocketContexts[0], SocketCount);
+    if (QUIC_FAILED(Status)) {
+        goto Exit;
+    }
```

Comment:
Is this universally supported? What happens if it's not?

Reply:
Probably a good point to not make this fail. Removed the status check, and added a note on why we're ignoring the status.